lib/jsdom/living/nodes/HTMLSelectElement-impl.js

```diff
@@ -90,6 +90,12 @@ class HTMLSelectElementImpl extends HTMLElementImpl {
     return this.options.length;
   }
 
+  get selectedOptions() {
+    return createHTMLCollection(this, () => domSymbolTree.treeToArray(this, { filter(node) {
```

Comment:
This is probably beyond my current knowledge about jsdom. I appreciate if you're willing to make this change.